parm/post_avblflds.xml

```diff
@@ -3665,7 +3665,7 @@
        <shortname>MAX_UPHL_ON_SPEC_HGT_LVL_ABOVE_GRND_2-5km</shortname>
        <longname>maximum Updraft Helicity on Specified Height Level Above Ground</longname>
        <pdstmpl>tmpl4_8</pdstmpl>
-       <pname>UPHL</pname>
+       <pname>MXUPHL</pname>
```

Comment:
This field is currently used by operational HRRR and regional FV3. It is better to keep unchanged.

Reply:
In the post_avblflds_raphrrr.xml file, they use MXUPHL for pname of UPP ID 420, so we had changed to match this for community wrf applications.  But if that has changed for newer operational HRRR and region FV3, then I can take this mod out of the PR and we will keep it in our seperate post_avblflds_comupp.xml.

Reply:
Kate,

You are right. I checked fv3sar_post_avblflds.xml and post_avblflds_raphrrr.xml, ID 420 has 
pname MXUPHL. Since gfs currently does not output this field. I think it is fine for changing
UPHL into MXUPHL. Thanks!